submissions/dashakim/oop-task/js/app.js

```diff
@@ -0,0 +1,114 @@
+const ENEMY = {
+  speed: 100,
+  varianceSpeed: 222,
+  location: [70, 155, 235]
+};
+const PLAYER = {
+  startX: 200,
+  startY: 404,
+  stepX: 101,
+  stepY: 83
+};
+
+const CANVAS = {
+  begin: -60,
+  end: 510,
+  leftSide: 0,
+  rightSide: 400,
+  top: 0,
+  bottom: 404,
+  halfCell: 50
+};
+
+var Character = function(x, y) {
+  this.x = x;
+  this.y = y;
+};
+Character.prototype.constructor = Character;
+Character.prototype.render = function() {
+  ctx.drawImage(Resources.get(this.sprite), this.x, this.y);
```

Comment:
Any property referred to in a class MUST be the class' own property or inherited from its parent. This class doesn't have any parents, so...